plugins/package-managers/node/src/main/kotlin/pnpm/ModuleInfo.kt

```diff
@@ -19,12 +19,20 @@
 
 package org.ossreviewtoolkit.plugins.packagemanagers.node.pnpm
 
+import java.io.ByteArrayInputStream
+
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.DecodeSequenceMode
 import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeToSequence
 
 private val JSON = Json { ignoreUnknownKeys = true }
 
-internal fun parsePnpmList(json: String): List<ModuleInfo> = JSON.decodeFromString(json)
+internal fun parsePnpmList(json: String): List<ModuleInfo> =
+    JSON.decodeToSequence<List<ModuleInfo>>(
+        ByteArrayInputStream(json.toByteArray()),
+        DecodeSequenceMode.WHITESPACE_SEPARATED
```

Comment:
Does the parser indeed use an incorrect mode if you leave it to auto-detection here?

Reply:
Yes, the auto-detection did not work, I had to set the mode explicitly.